independent-publisher-connectors/MS Graph Groups and Users/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1151 @@
+              "\"request-id\"": {
+                "description": "\"request-id\"",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "List Direct Group Members",
+        "operationId": "ListDirectGroupMembers",
+        "description": "Retrieve direct members of a group with count",
+        "parameters": [
+          {
+            "name": "group-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Object ID of the AAD group",
+            "x-ms-summary": "Object ID of the AAD group",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "jobTitle ne null",
+            "description": "Filter By",
+            "x-ms-summary": "Filter By"
+          },
+          {
+            "name": "$select",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "displayName,userPrincipalName,id,jobTitle,mailNickname",
+            "description": "Select which columns to retrieve",
+            "x-ms-summary": "Data point (column) selection"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "required": true,
+            "type": "boolean",
+            "default": true,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "ConsistencyLevel",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "eventual",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/v1.0/users/{id}/licenseDetails": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "The operation was successful.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "@odata.context": {
+                  "type": "string",
+                  "description": "The Odata.context link.",
+                  "title": "Odata Context"
+                },
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "id",
+                        "title": "ID"
+                      },
+                      "skuId": {
+                        "type": "string",
+                        "description": "skuId",
+                        "title": "Sku ID"
+                      },
+                      "skuPartNumber": {
+                        "type": "string",
+                        "description": "Name of the sku",
+                        "title": "Sku Part Number"
```

Comment:
@jagjani03 if you use abbreviations, please make sure to capitalize them to make it clear.

Reply:
@v-EgorKozhadei This has been completed. Thanks!